assets/js/camptix-multi-popup.js

```diff
@@ -1,218 +1,128 @@
-            $('.message').text('Please Enter correct Mobile Number Or Number with STD Code');
-            $('.message').css('color','red');
-            $('.mobile').val('');
-            $('.mobile').focus();
-            //alert();
-            e.preventDefault();
-
-		return false;
-
-        } 
-     
-		// Bailout.
-		if (camptix_inr_vars.gateway_id !== $('select[name="tix_payment_method"]', $form).val()) {
-			return true;
-		}
-
-		e.preventDefault();
-
-		return false;
-	});
-
-	/**
-	 * When the submit button is clicked.
-	 */
-	$form.on('click touchend', 'input[type="submit"]', function (e) {
-		// Bailout.
-		if (camptix_inr_vars.gateway_id !== $('select[name="tix_payment_method"]', $form).val()) {
-			return true;
-		}
-
-		e.preventDefault();
-
-		// Validate custom attendee information fields.
-		if (!validate_fields()) {
-			show_errors('<div class="tix-error">' + camptix_inr_vars.errors.phone + '</div>');
-
-			return false;
-		}
-
-		var $submit_button = $(this),
-			$response;
-
-		$.post($form.attr('action'), $form.serialize())
-			.done(function (response) {
-				// Bailout.
-				if (!response.success) {
-					var $el = $('<div></div>').html(response);
-
-					show_errors($('#tix-errors', $el).html());
-
-					return false;
-				}
-
-				// Cache response for internal use in Razorpay.
-				$response = response;
-
-				razorpay_handler = new Razorpay({
-					'key'     : camptix_inr_vars.merchant_key_id,
-					'order_id': order_id,
-					'name'    : $response.data.popup_title,
-					'image'   : camptix_inr_vars.popup.image,
-					// 'description' : '',
-					'handler' : function (response) {
-						// Remove loading animations.
-						// $form.find('.give-loading-animation').hide();
-						// Disable form submit button.
-						$submit_button.prop('disabled', true);
-
-						// Submit form after charge token brought back from Razorpay.
-						// Redirect to success page.
-						window.location.assign($response.data.return_url + '&transaction_id=' + order_id + '&receipt_id=' + receipt_id);
-					},
-
-					// You can add custom data here and fields limited to 15.
-					// 'notes': {
-					// 'extra_information' : $response.data
-					// },
-					'prefill': {
-						'name'   : $response.data.fullname,
-						'email'  : $response.data.email,
-						'contact': $response.data.phone
-					},
-
-					'modal': {
-						'ondismiss': function () {
-							// Remove loading animations.
-							$form.find('.give-loading-animation').hide();
-
-							// Re-enable submit button and add back text.
-							$submit_button.prop('disabled', false);
-						}
-					},
-
-					'theme': {
-						'color'        : camptix_inr_vars.popup.color,
-						'image_padding': false
-					}
-				});
-
-				razorpay_handler.open();
-			})
-			.fail(function () {
-			})
-			.always(function () {
-				// Enable form submit button.
-				$submit_button.prop('disabled', false);
-			});
-
-		return false;
-	});
-});
+/**
+ * Give - Razorpay Popup Checkout JS

```

Comment:
Comment needs update as this common file for gateways not specific to "Razorypay"